test/bats/tests/aws/BasicTestMountSPC.yaml

```diff
@@ -12,18 +12,18 @@ spec:
       key: username
   parameters:
     objects: | 
-        - objectName: "ParameterStoreTest1"
+        - objectName: "ParameterStoreTest1-${UUID}"
           objectType: "ssmparameter"
-        - objectName: "ParameterStoreTestWithLongName"
+        - objectName: "ParameterStoreTestWithLongName-${UUID}"
           objectAlias: "ParameterStoreTest2"
           objectType: "ssmparameter"
-        - objectName: "ParameterStoreRotationTest"
+        - objectName: "ParameterStoreRotationTest-${UUID}"
           objectType: "ssmparameter"
-        - objectName: "SecretsManagerRotationTest"
+        - objectName: "SecretsManagerRotationTest-${UUID}"
           objectType: "secretsmanager"
-        - objectName: "SecretsManagerTest1"
+        - objectName: "SecretsManagerTest1-${UUID}"
```

Comment:
Maybe just use `$SM_TEST_1_NAME` instead of a constant + variable? Similar changes for all the `objectName` in this file.

Reply:
yup that would be cleaner